src/frontend/staff-portal/src/app/services/disputes.service.ts

```diff
@@ -167,6 +167,37 @@ export class DisputesService implements IDisputesService {
       );
   }
 
+    /**
+     * Put the dispute to RSI by Id.
+     *
+     * @param disputeId
+     */
+     public validateDispute(disputeId: string): Observable<Dispute> {
+
+      return this.disputeService.apiDisputeDisputeIdValidatePut(disputeId)
+        .pipe(
+          map((response: any) =>
```

Comment:
map((response: any) =>
           this.logger.info('DisputesService::putDispute', response)
           return  response ? response : null;
          ),